composer.json

```diff
@@ -57,29 +57,17 @@
             "homepage": "http://dynasource.eu"
         }
     ],
+    "minimum-stability": "dev",
     "require": {
-        "yiisoft/yii-base-web": "^3.0"
```

Comment:
As discussed here https://github.com/yiisoft/yii-base-web/issues/45#issuecomment-415479940 - you do not know which kind of app to user wants to create. Why should we assume that there's always a web application - in fact that's the most complicated one, since it also requires a specific asset installation system.

But the installation should be scriptable or even interactive in a later stage.